extensions/vertx-http/deployment/src/test/resources/conf/http-permission-grant-config.properties

```diff
@@ -0,0 +1,42 @@
+quarkus.http.auth.basic=true
+quarkus.http.auth.policy.t1.roles-allowed=test,test2,admin,other
```

Comment:
I think that having `quarkus.http.auth.policy.t1.roles-allowed=test,test2,admin,other` should not be necessary for the permissions based access control to work.

Something like `quarkus.http.auth.policy.t1.permissions.test=create-test` should be enough, otherwise we have 2 access controls, the 2nd one derived from the 1st one, at the same time, and I'm not sure this is what is necessary, now, it can work, but expressing a `roles-allowed` requirement should not be a prerequisite.

Reply:
Lets imagine we have a token with roles, so OIDC will create SecurityIdentity with those roles, we have the input for the mapping to permission ready now, RBAC (`roles-allowed=`) is arguably coarser grained, really about `roles` like `admin` vs permissions like `read`, so expressing RBAC should be totally optional IMHO

Reply:
okay @sberyozkin I re-read what you wrote and I think I misunderstood you at first. `quarkus.http.auth.policy.t1.permissions.test=create-test` represents tole `test` mapped to `create-test` and that is it, right? That can be done by setting `roles-allowed=**` by default. I'll do and nothing else as that makes sense to me and that's how I understand you.